utils/funcexp/func_ceil.cpp

```diff
@@ -592,6 +575,10 @@ IDB_Decimal Func_ceil::getDecimalVal(Row& row,
                     // decimal point so that we return the largest integer value not less than X.
                     if ((tmp - (ret.s128Value * scaleDivisor)) > 0)
                         ret.s128Value += 1;
+*/
+                    ret = IDB_Decimal(ret.getRoundedIntegralPart(),
+                                      ret.scale,
+                                      ret.precision);
```

Comment:
Why did you put the old code into a comment rather than just delete it?

Reply:
Missed that part.

Reply:
Removed.